components/movementsensor/mpu6050/mpu6050.go

```diff
@@ -1,4 +1,21 @@
-// Package mpu6050 implements the movementsensor interface for an MPU-6050 6-axis accelerometer.
+// Package mpu6050 implements the movementsensor interface for an MPU-6050 6-axis accelerometer. A
+// datasheet for this chip is at
+// https://components101.com/sites/default/files/component_datasheet/MPU6050-DataSheet.pdf and a
+// description of the I2C registers is at
+// https://download.datasheets.com/pdfs/2015/3/19/8/3/59/59/invse_/manual/5rm-mpu-6000a-00v4.2.pdf
+//
+// We support reading the accelerometer, gyroscope, and thermometer data off of the chip. We do not
+// yet support using the digital interrupt pin to notify when interesting things happen (freefall,
+// collisions, etc.), nor do we yet support using the I2C slave connection to add an external clock
```

Comment:
Eh... although this is the language used in the data sheet, we should probably shy away from that phrasing and just call it the "external I2C connection" instead.